src/main.c

```diff
@@ -1694,7 +1694,11 @@ getout(int exitval)
     }
 
 #ifdef FEAT_VIMINFO
-    if (*p_viminfo != NUL)
+    if (
+# if defined(EXITFREE) || defined(PROTO)
+	    entered_free_all_mem == FALSE &&
+# endif
+	    *p_viminfo != NUL)
```

Comment:
:memo: When leak detect is enabled, `p_viminfo` is accessed after freed.

Reply:
ah thanks